_data/authors.yml

```diff
@@ -169,4 +169,9 @@ vitalii_slobodianyk:
   avatar: vitalii_slobodianyk.svg
   github: slovit
   is_alumni: true
-  
+grant_guo:
+  name: Grant Guo
+  position: Senior Software Developer
+  info: passionate back-end developer interested in applying distributed data computation software to domains, keep curiosity on new things 
```

Comment:
Capitalize `Passionate`.

Also for grammatical purposes, you might want to reword:
"... , keep curiosity on new things" to something like "..., forever curious about news things."